### ISSUE TITLE
How do I permanently set the Dump1 tab to show my EXE's ".data" section?

### ISSUE THREAD

**Issue by Videogamer555** [x64dbg/x64dbg#2721]
Am I missing something here? I can't seem find a way to configure the Dump1 tab to automatically show the ".data" section of my EXE or DLL file. I know OllyDbg does this by default. I don't know if there's some configuration setting I'm just overlooking in x64dbg or what. Please kindly help me figure out how to do this. And if it's not currently possible, please consider adding this as a feature in a future version of the software.

**Comment by mrexodia** [comment]
Feel free to open a pull request to address your issue.

**Comment by Videogamer555** [comment]
> 
> 
> Feel free to open a pull request to address your issue.

Is there some way to do it currently in the settings menu? Or am I overlooking something? Please let me know this first, before I set off to try to fix something that maybe doesn't need fixing.

**Comment by torusrxxx** [comment]
No, there isn't such a setting because nobody has written the code for it.
